HumanoidCtrl:
  ros__parameters:
    InputTopicName: "/HumanoidCtrl/humanoid_msgs/msg/JointState"
    JointTopic: "/JointState"
    PublishRate: 25.0
    RobotType: "Ti5Robot"
    # RobotType: "CrpRobot"
    # RobotType: "UnitreeG1Dof29"
    # RobotType: "UnitreeG1Dof23"
    
    